### PR TITLE
Code commenter

### PR DESCRIPTION
**Pull request by daileyj93**
Modified the script commenter by @CynicRus (PR#474) to be compatible with the "undo" command.
Comments or uncomments entire selection of code.
Ctrl + / to comment or uncomment slection - similar to Lazarus IDE.

### CODE REVIEW

**Comment by ollydev** [comment]
Does this only comment from the start of the line? It would be more in line with other IDEs (and Lazarus) if it comments vertically from the start X of the block.

File: Projects/Simba/simbaunit.pas [hunk 1]
```diff
@@ -284,7 +285,8 @@   TSimbaForm = class(TForm)
     ToolButton8: TToolButton;
     MTrayIcon: TTrayIcon;
     FunctionListHint: THintWindow;
-    procedure ActionColorsExecute(Sender: TObject);                                            
+    procedure ActCodeCommentExecute(Sender: TObject);
```
Comment (smithtrenton) [inline Projects/Simba/simbaunit.pas]: Why not keep naming consistent?

**Comment by MerlijnWajer** [comment]
Why was this closed?

**Comment by daileyj93** [comment]
I closed it because I screwed up the branch. I tried to rebase, but it totally corrupted the commit history. I guess you are not supposed to force push to an open PR. I re-opened this on PR#480. I hope this is okay.

**Pull request closed without merging**